README.md

```diff
@@ -71,6 +72,18 @@ Choose what you find more suitable for you:
 - [Extreme Rate Shells](https://extremerate.com/collections/steam-deck-shells) - Extreme Rate Steam Deck Shells.
 - [GamingCobra Shells](https://www.gamingcobra.com/collections/steam-deck-shells) - Gaming Cobra Steam Deck Shells.
 - [JSAUX Transparent Back Plate](https://jsaux.com/products/transparent-back-plate-for-steam-deck-pc0106) - JSAUX Transparent Back Plate for Steam Deck.
+- [Corsair K70 Pro Mini](https://www.amazon.com/dp/B0B4SW81GW/) - 60% wireless mechanical keyboard.
+- [Razer Viper V2 Pro](https://www.amazon.it/dp/B09QFVSYX6) - Premium gaming mouse with great set of features.
```

Comment:
I've tried the .com link for this and seems to be linking towards the same product.
The only thing that concerns me is that is currently inavailable.
Do you want to keep it with a .com link or remove it?

Reply:
Updated with another link of amazon.com